website/docs/language/expressions/custom-conditions.mdx

```diff
@@ -203,6 +204,15 @@ You should also consider the following questions when creating preconditions and
 - Which approach is more convenient? If a particular resource has many dependencies that all make an assumption about that resource, it can be pragmatic to declare that once as a post-condition of the resource, rather than declaring it many times as preconditions on each of the dependencies.
 - Is it helpful to declare the same or similar conditions as both preconditions and postconditions? This can be useful if the postcondition is in a different module than the precondition because it lets the modules verify one another as they evolve independently.
 
+## Check Assertions
+
+-> **Note:** Check blocks and their assertions are only available in Terraform v1.5.0 and later.
+
```

Comment:
Can we add more flavor text to the beginning of this? A broader, "this is when and why you use checks" paragraph?

Reply:
Also please mention when checks are actually run!

Reply:
I think this is the perfect amount of context, thank you so much for adding it!

My general rule of thumb is usually, "Repeat information **if** someone needs more context to understand what we are talking about on _this_ page." I like to keep things as DRY as possible, but also ensure folks have enough context _without_ having to open to a new page. It's _definitely_ a thin line to walk though 😂